src/main/java/frc/robot/RobotContainer.java

```diff
@@ -245,20 +252,20 @@ public void initAuton(Constants.Auton mode) {
     boolean success = true;
     switch (mode) {
       case TEST:
-        if (driveBaseSubsystem != null) {
+        /* if (driveBaseSubsystem != null) {
           autonModes = new AutonModes(driveBaseSubsystem);
         } else {
           success = false;
         }
-        break;
+        break;*/
       case PUSHTAXI: // the two taxis
       case INTAKETAXI:
-        if (driveBaseSubsystem != null && intakeSubsystem != null && cdsSubsystem != null) {
+        /* if (driveBaseSubsystem != null && intakeSubsystem != null && cdsSubsystem != null) {
           autonModes = new AutonModes(driveBaseSubsystem, intakeSubsystem, cdsSubsystem);
         } else {
           success = false;
         }
-        break;
+        break;*/
```

Comment:
I'm assuming that the code was commented out to try and optimize the run time?

The cycles saved are minimal if any.    Don't need to change it, but in this case, I would have optimized for clarity instead of speed.